bukuserver/templates/bukuserver/home.html

```diff
@@ -18,29 +18,29 @@
 
 {% block body %}
 {{ super() }}
-<main class="container">
-  <div style="padding: 40px 15px; text-align: center;">
+<main class="container text-center">
+  <div style="padding: 40px 15px">
     <h1>BUKU</h1>
     <p class="lead">Bookmark manager like a text-based mini-web</p>
     <p>
-      <a class="btn btn-lg btn-success" href="{{url_for('bookmark.index_view')}}" role="button">Bookmarks</a>
-      <a class="btn btn-lg btn-success" href="{{url_for('tag.index_view')}}" role="button">Tags</a>
-      <a class="btn btn-lg btn-success" href="{{url_for('statistic.index')}}" role="button">Statistics</a>
+      <a class="btn btn-lg btn-success" href="{{ url_for('bookmark.index_view') }}" role="button">Bookmarks</a>
+      <a class="btn btn-lg btn-success" href="{{ url_for('tag.index_view') }}" role="button">Tags</a>
+      <a class="btn btn-lg btn-success" href="{{ url_for('statistic.index') }}" role="button">Statistics</a>
     </p>
-    <div class=" col-md-4 col-md-offset-4">
-      <form class="form-horizontal" action="{{url_for('admin.search')}}" method="POST">
+    <div class="col-md-4 col-md-offset-4">
+      <form action="{{ url_for('admin.search') }}" method="POST">
         <div class="form-group">
-          {{form.keyword.label}}
-          {{form.keyword()}}
+          {{ form.keyword.label }}
+          {{ form.keyword(class_='form-control', style='display: inline;  width: auto') }}
         </div>
-        <div class="text-left col-sm-offset-2">
-          <div class="form-group"> {{form.deep()}} {{form.deep.label}} </div>
-          <div class="form-group"> {{form.regex()}} {{form.regex.label}} </div>
+        <div class="text-left col-sm-offset-3">
+          <div class="checkbox"> {{form.deep()}} {{form.deep.label}} </div>
+          <div class="checkbox"> {{form.regex()}} {{form.regex.label}} </div>
```

Comment:
Applying the Bootstrap theme, with minimum layout changes
![main search form](https://github.com/user-attachments/assets/8152d2b0-0f18-4f4f-b62e-ce31d6824407)